app/models/workflow/convert_legacy_extractors_config.rb

```diff
@@ -0,0 +1,40 @@
+class Workflow::ConvertLegacyExtractorsConfig
+  attr_reader :workflow
+
+  def initialize(workflow)
+    @workflow = workflow
+  end
+
+  def update(config)
+    return if config.nil?
+
+    workflow.extractors.where.not(key: config.keys).delete_all
+
+    config.each do |key, config|
+      extractor = workflow.extractors.find_by(key: key)
+      extractor ||= extractor_type(config).new(workflow: workflow, key: key)
+      extractor.config = config.except("min_version", "type")
```

Comment:
https://github.com/zooniverse/caesar/pull/130/files#diff-c954f372b944f5a449c3761b6b6dd1f6L14

This came from `config["minimum_version"]` not min_version.